intake/config.py

```diff
@@ -35,6 +35,7 @@
     "extra_imports": [],
     "import_block_list": [],
     "reader_avoid": [],
+    "raise_on_error": False,
```

Comment:
environment_conf_parse_error with possible values "raise", "warn", "ignore" with the latter as default?